src/Scouting.elm

```diff
@@ -1,141 +1,198 @@
-            [ Html.input
-                [ placeholder "Scouter's name..."
-                , onInput NameInput
-                , value model.scouterName
-                ]
-                []
-            , Html.input
-                [ placeholder "Team's number..."
-                , onInput TeamInput
-                , value <|
-                    if 0 == model.teamNum then
-                        ""
-
-                    else
-                        String.fromInt model.teamNum
-                ]
-                []
-            , Html.input
-                [ placeholder "Match number..."
-                , onInput MatchInput
-                , value <|
-                    if 0 == model.matchNum then
-                        ""
-
-                    else
-                        String.fromInt model.matchNum
-                ]
-                []
-            , Html.input
-                [ style "visibility" dropDown_driverStation
-                , placeholder "Driver station..."
-                , onInput DriverStationInput
-                , value model.driverStation
-                ]
-                []
-            , Html.button [ onClick Start ]
-                [ Html.text <|
-                    if model.isStarted then
-                        "Started"
-
-                    else
-                        "Startn't"
-                ]
-            ]
-        , Html.div []
-            [ Html.text <|
-                "Status:  name - "
-                    ++ model.scouterName
-                    ++ ", team - "
-                    ++ String.fromInt model.teamNum
-                    ++ ", match - "
-                    ++ String.fromInt model.matchNum
-                    ++ ", station - "
-                    ++ model.driverStation
-            ]
-        ]
+module Scouting exposing (Model, Msg, driverStations, init, main, update, view)
+
+import Browser
+import Html
+import Html.Attributes exposing (placeholder, style, value)
+import Html.Events exposing (onClick, onInput)
+import List.Extra exposing (elemIndex, getAt)
+import Maybe exposing (withDefault)
+import Maybe.Extra exposing (unwrap)
+import Tuple exposing (first, second)
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+type Msg
+    = NameInput String
+    | TeamInput String
+    | MatchInput String
+    | Start
+
+
+type alias Model =
+    { scouter : String
+    , team : Maybe Int
+    , match : Maybe Int
+    , driverStation : String
+    , isStarted : Bool
+    }
+
+
+driverStations : List (List Int)
+driverStations =
+    [ [ 1690, 1574, 3339, 254, 2056, 1323 ]
+    , [ 118, 1577, 1024, 2056, 1690, 254 ]
+    , [ 1574, 3339, 1577, 1323, 1024, 118 ]
+    , [ 3339, 1574, 1577, 1024, 118, 2056 ]
+    ]
+
+
+init : Model
+init =
+    { scouter = ""
+    , team = Nothing
+    , match = Nothing
+    , driverStation = ""
+    , isStarted = False
+    }

```

Comment:
See https://github.com/orbit1690/Scouting-2019/pull/4#discussion_r353351967 - you don't need to define `init` from the ground up - you've already got `Model`.

Reply:
I didn't understand how to

Reply:
Remind me on Friday 🙂